src/main/java/edu/ucsb/cs156/courses/services/UCSBCurriculumService.java

```diff
@@ -60,6 +60,8 @@ public UCSBCurriculumService(RestTemplateBuilder restTemplateBuilder) {
 
     public static final String SECTION_ENDPOINT = "https://api.ucsb.edu/academics/curriculums/v1/classsection/{quarter}/{enrollcode}";
 
+    public static final String ALL_SECTIONS_ENDPOINT = "https://api.ucsb.edu/academics/curriculums/v3/classes/{quarter}/{enrollcode}";
```

Comment:
What distinguishes SECTION_ENDPOINT from ALL_SECTIONS_ENDPOINT?  I'm confused.

Comments are not usually encouraged in this style of code--we prefer the code to be self-documenting through variable name choice.  However, this might be an exception, where a comment or two would be really helpful.